specification/frontdoor/resource-manager/Microsoft.Network/preview/2019-03-01-preview/examples/WafListPolicies.json

```diff
@@ -1,6 +1,6 @@
 {
     "parameters": {
-        "api-version": "2018-08-01",
+        "api-version": "2019-03-01",
```

Comment:
API changes have already been reviewed in the private repo ([**PR 690**](https://github.com/Azure/azure-rest-api-specs-pr/pull/690)).